### PR TITLE
Vet clinic database: database performance audit

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
### Hello there ✌ in this part we were working together to decrease the execution time .

 _Bellow some of the Project requirements:_

- Find a way to decrease the execution time of the first query ✔
-  While you are making changes, check if they help by running EXPLAIN ANALYZE. Once you succeed, take a screenshot of the EXPLAIN ANALYZE result showing that time actually decreased ✔
- Any changes you made to your database schema should be added to the schema.sql✔
- In your pull request description include screenshots with EXPLAIN ANALYZE results ✔

**SCREENSHOTS**

# visits where id_animal = 4;
## Before
image
## After
image

# visits where id_vet = 2;
## Before
image
## After 
image

# owners where email = 'owner_18327@mail.com';
## Before
image
## After
image

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -181,4 +181,12 @@ LEFT JOIN animals ON animals.id_animals = visits.id_animal
 LEFT JOIN species ON animals.id_species = species.id_species
 LEFT JOIN vets ON vets.id_vets = visits.id_vet
 WHERE vets.name_vets = 'Maisy Smith'
-GROUP BY species.name;
\ No newline at end of file
+GROUP BY species.name;
+
+CREATE INDEX idx_visits_vet_id ON visits(id_vet);
+CREATE INDEX ON visits (id_animal);
+CREATE INDEX ON owners ((lower(email)));
```
Comment (SajjadAhmad14) [inline queries.sql]: Please add these queries (add indexes to foreign keys) to `schema.sql` file.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Elim, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
README is professional. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 1 above)
Comment (CarlosE-Feliz) [reply queries.sql]: Good to know, thanks for noticing it I just forgot to move them into `schema.sql`

**Comment by CarlosE-Feliz** [comment]
# Note for the next reviewer 😎✌

As the previous reviewer mentioned, I added the indexs to the `schema.sql`

image

Hopping now we can move to the next part and keep coding!

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @CarlosE-Feliz 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**